docs/reference/garbage-collection.md

```diff
@@ -283,11 +283,19 @@ This utility is a Spark application that uses [distCp](https://hadoop.apache.org
 * Backup: copy expired objects from your repository's storage namespace to an external location before running GC in [sweep only mode](#sweep-only-mode).  
 * Restore: copy objects that were hard-deleted by GC from an external location you used for saving your backup into your repository's storage namespace.
 
+### Job options
+
+- This will only work with Hadoop versions > 3.1.3, meaning only for our hadoop 3 client.
+- [https://docs.cloudera.com/HDPDocuments/HDP3/HDP-3.1.5/administration/content/distcp_faq.html](https://docs.cloudera.com/HDPDocuments/HDP3/HDP-3.1.5/administration/content/distcp_faq.html) explain distCp performance limitations and workarounds.
+- Configuration options and link to docs
+- Note that the utility is not fast due to distcp limitations. you may prefer to backup your whole storage namespace with azcopy/ aws cp / rclone.
```

Comment:
```suggestion
- Note that the utility is not fast due to distcp limitations. you may prefer to backup your whole storage namespace with azcopy / aws cp / [rclone](https://rclone.org/).
```
Perhaps link to all of these?

Reply:
added. I accidentally left this docs change here, so I only left what's relevant now and will add more when I'll release https://github.com/treeverse/lakeFS/pull/4378.